pygsti/baseobjs/basis.py

```diff
@@ -1782,283 +1760,3 @@ def create_simple_equivalent(self, builtin_basis_name=None):
                 if all([c.name == first_comp_name for c in self.component_bases]):
                     builtin_basis_name = first_comp_name  # if all components have the same name
         return BuiltinBasis(builtin_basis_name, self.elsize, sparse=self.sparse)
-
-
-class EmbeddedBasis(LazyBasis):
```

Comment:
No action items, just a personal note that we are removing this so that I can include it in my spring cleaning section of the release notes.